examples/safety_torque_off.py

```diff
@@ -17,8 +17,14 @@ def main(interface_ip, slave_id, dict_path):
     # Create and start the FSoE master handler
     mc.fsoe.create_fsoe_master_handler()
     mc.fsoe.start_master(start_pdos=True)
-    # Deactivate the STO and enable the motor
+    # Wait for the master to reach the Data state
+    mc.fsoe.wait_for_state_data()
```

Comment:
Don't you want to set a timeout on the example?

Reply:
I don't think it's necessary.